onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -415,57 +415,6 @@ def _sum_input_wrangler(
     return args, kwargs
 
 
-def _upsample_bilinear2d_input_wrangler(
-    args: list[Any], kwargs: dict[str, Any]
-) -> tuple[list[Any], dict[str, Any]]:
-    # Wrangler for the signature difference between
-    #   'nn.functional.upsample_bilinear'
-    # and
-    #   'aten::upsample_bilinear2d'
-    # https://pytorch.org/docs/stable/generated/torch.nn.functional.upsample_bilinear.html
-    if "size" in kwargs:
-        args.append(np.array(kwargs["size"], dtype=np.int64))
-        del kwargs["size"]  # promote tensor type kwargs to args
-    else:
-        args.append(None)
-    if "align_corners" in kwargs:
-        args.append(kwargs["align_corners"])
-        del kwargs["align_corners"]
-    else:
-        args.append(True)  # Fill in the default value
```

Comment:
False was not tested? @BowenBao

Reply:
It is on pytorch side, that was the failing case leading to my fix earlier.

It was not tested on onnxscript side.